<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabDiveStatistics</class>
 <widget class="QWidget" name="TabDiveStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>286</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2a">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>

         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBoxb">
           <property name="title">
            <string>Max. depth</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="statsDepthLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="MinMaxAvgWidget" name="depthLimits" native="true"/>
            </item>
           </layout>
          </widget>
         </item>

         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_14">
           <property name="title">
            <string>Duration</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="statsDurationLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="MinMaxAvgWidget" name="timeLimits" native="true"/>
            </item>
           </layout>
          </widget>
         </item>

         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBox_8b">
           <property name="title">
            <string>Temperature</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="statsTempLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="MinMaxAvgWidget" name="tempLimits" native="true"/>
            </item>
           </layout>
          </widget>
         </item>

         <item row="0" column="3">
          <widget class="QGroupBox" name="groupBox_4b">
           <property name="title">
            <string>SAC</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="statsSacLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="MinMaxAvgWidget" name="sacLimits" native="true"/>
            </item>
           </layout>
          </widget>
         </item>

         <item row="1" column="0">
          <widget class="QLabel" name="emptyline">
           <property name="text">
            <string></string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeft</set>
           </property>
          </widget>
         </item>

         <item row="2" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBoxa1">
           <property name="title">
            <string>All selected dives</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>

            <item row="0" column="0">
             <widget class="QLabel" name="totalDivesLabel">
              <property name="text">
               <string>Total number of dives selected :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeft</set>
              </property>
             </widget>
            </item>

            <item row="0" column="1">
             <widget class="QLabel" name="divesAllText">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeft</set>
              </property>
             </widget>
            </item>

            <item row="1" column="0">
             <widget class="QLabel" name="totalTimeLabel">
              <property name="text">
               <string>Total duration of selected dives :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeft</set>
              </property>
             </widget>
            </item>

            <item row="1" column="1">
             <widget class="QLabel" name="totalTimeAllText">
              <property name="text">
               <string></string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeft</set>
              </property>
             </widget>
            </item>

            <item row="2" column="0">
             <spacer name="verticalSpacer_1">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>5</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>

           </layout>
          </widget>
         </item>

         <item row="2" column="2" rowspan="2" colspan="2" >
          <widget class="QGroupBox" name="groupBox_13">
           <property name="title">
            <string>Gas consumption</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="statsGasConsumptionLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>

            <item>
             <widget class="QScrollArea" name="scrollArea_14">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>

              <widget class="QLabel" name="gasConsumption">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignTop</set>
               </property>
              </widget>

             </widget>    
            </item>
           </layout>
          </widget>
         </item>

         <item row="3" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>

        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>

 <customwidgets>
  <customwidget>
   <class>MinMaxAvgWidget</class>
   <extends>QWidget</extends>
   <header>desktop-widgets/tab-widgets/TabDiveStatistics.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
